content/applications/productivity/iot/config/updating_iot.rst

```diff
@@ -0,0 +1,132 @@
+
+   .. seealso::
+      :ref:`flash_sdcard/etcher`.
+
+.. important::
+   An :guilabel:`Upgrade` button will appear if the :abbr:`IoT (Internet of Things)` box is on the
+   latest image. In this case, the :abbr:`IoT (Internet of Things)` box will instead update the core
+   code (without changing the image itself. In other words this button forces Odoo based :abbr:`IoT
+   (Internet of Things)` box code to update with the latest version from GitHub. This can be done
+   **without** an Odoo database to synchronize with.
+
+.. image:: updating_iot/flash-upgrade.png
+   :align: center
+   :alt: IoT box software upgrade in the IoT Box Home Page.
+
+.. _flash_sdcard/etcher:
+
+Flashing the SD card on IoT box
+===============================
+
+In some circumstances, the :abbr:`IoT (Internet of Things)` box's micro SD Card may need to be
+re-flashed with Etcher software to benefit from Odoo's latest :abbr:`IoT (Internet of Things)` image
+update. This means that the Odoo :abbr:`IoT (Internet of Things)` box software may need to be
+updated in instances of a new :abbr:`IoT (Internet of Things)` box, or when a handlers update and
+update from the :abbr:`IoT (Internet of Things)` box home page does not fix issues.
+
+.. warning::
+   It is often imperative to re-flash the :abbr:`IoT (Internet of Things)` box's image after
+   upgrading the Odoo database to a new version.
+
+.. note::
+   A computer with a micro SD card reader/adapter is required to re-flash the micro SD card.
+
+Navigate to Balena's website and download `Etcher <https://www.balena.io/>`_. It is a free and
+open source utility used for burning image files onto drives. Click to `download
+<https://www.balena.io/etcher#download-etcher>`_. Install and launch the program on the computer.
+
+Then download the latest IoT image from `nightly <http://nightly.odoo.com/master/iotbox/>`_ :
+`iotbox-latest.zip`
+
+This image is compatible with all supported versions of Odoo.
+
+The images should be downloaded and extracted to a convenient file location.
+
+After this step is complete, insert the :abbr:`IoT (Internet of Things)` box's micro SD card into
+the computer or reader. Open *Etcher* and select :guilabel:`Flash from file`, then find and select
+the image just downloaded and extracted. Next, select the drive the image should be burned to.
+Lastly, click on :guilabel:`Flash` and wait for the process to finish.
+
+.. image:: updating_iot/etcher-app.png
+   :align: center
+   :alt: Balena's Etcher software dashboard.
+
+.. tip::
+   Balena's Etcher software also allows for the administrator to flash the :abbr:`SD (Secure
+   Digital)` SD card from an :abbr:`URL (Uniform Resource Locator)`. To flash from an :abbr:`URL
+   (Uniform Resource Locator)`, simply click :guilabel:`Flash from URL` in stead of :guilabel:`Flash
+   from file`. The option to authenticate with a secure server using a :guilabel:`username` and
+   :guilabel:`password` is available.
```

Comment:
we don't use "username" and "password" so maybe precise that it should be left empty.

Also maybe precise the URL of the latest image as it will always be the same:
https://nightly.odoo.com/master/iotbox/iotbox-latest.zip